services/ai/run.ps1

```
@@ -0,0 +1,8 @@
+$ErrorActionPreference = "Stop"
+
+if (-not (Test-Path ".env")) {
+    Copy-Item ".env.example" ".env"
+    Write-Host "Created .env from .env.example. Add OPENAI_API_KEY before using LLM responses." -ForegroundColor Yellow
```

PSScriptAnalyzer (1.24.0)

[warning] File 'run.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)